src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActiveState.test.js

```diff
@@ -148,4 +153,28 @@ describe('elements/content-sidebar/ActiveState/activity-feed/ActiveState', () =>
         const wrapper = shallow(<ActiveState inlineError={activityFeedError} items={[]} currentUser={currentUser} />);
         expect(wrapper).toMatchSnapshot();
     });
+
+    test.each`
+        currentFileVersionId | isCurrentVersion
+        ${'123'}             | ${true}
+        ${'456'}             | ${false}
+    `(
+        'should correctly reflect annotation activity isCurrentVersion as $isCurrentVersion based on file version id as $currentFileVersionId',
+        ({ currentFileVersionId, isCurrentVersion }) => {
+            const wrapper = shallow(
```

Comment:
I know this is an existing test file, but would a `getWrapper` helper be worthwhile to avoid redeclaring all the base props for each test?